proxy/grpcproxy/kv.go

```diff
@@ -81,8 +81,17 @@ func (p *kvProxy) Txn(ctx context.Context, r *pb.TxnRequest) (*pb.TxnResponse, e
 	return (*pb.TxnResponse)(resp), err
 }
 
+func (p *kvProxy) Compact(ctx context.Context, r *pb.CompactionRequest) (*pb.CompactionResponse, error) {
+	resp, err := p.client.Do(ctx, CompactionRequestToOp(r))
```

Comment:
why not just use https://github.com/coreos/etcd/blob/master/clientv3/kv.go#L101?